mirrord/config/src/ci.rs

```diff
@@ -0,0 +1,54 @@
+use std::path::PathBuf;
+
+use mirrord_analytics::CollectAnalytics;
+use mirrord_config_derive::MirrordConfig;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::config::source::MirrordConfigSource;
+
+/// Configuration for mirrord for CI.
+///
+/// Logs from the app are sent to `/dev/null` when using `mirrord ci start`. You can pipe them
+/// to a file by setting one of the `stdio` file paths (note that the file doesn't have to exist,
+/// mirrord will take care of creating it).
+///
+/// ```json
+/// {
+///   "ci": {
+///     "stdout_file": "/tmp/mirrord-ci-logs/stdout",
+///     "stdin_file": "/tmp/mirrord-ci-logs/stdin",
+///     "stderr_file": "/tmp/mirrord-ci-logs/stderr"
+///   }
+/// }
+/// ```
+#[derive(MirrordConfig, Clone, Debug, Serialize, Deserialize, PartialEq, Default)]
+#[config(map_to = "CiFileConfig", derive = "JsonSchema")]
+#[cfg_attr(test, config(derive = "PartialEq, Eq"))]
+pub struct CiConfig {
```

Comment:
I am not sure this is future-proof enough. Maybe we just need an optional output directory path, and default to a tmp dir if none. The directory is not limited to logs, we might need it for other stuff like tracking process status etc. in the future.
For each client app process (run by ci start), we create a subfolder (named by the binary + unique suffix) for now and writes output and error to `stdout.log` and `stderr.log`.
WDYT?